src/main/resources/i18n/Validation_de.properties

```diff
@@ -76,7 +76,7 @@ VariableDefaultValuesOfCorrectType.badDefault=Validierungsfehler ({0}) : Ung
 #
 VariablesAreInputTypes.wrongType=Validierungsfehler ({0}) : Eingabevariable ''{1}'' Typ ''{2}'' ist kein Eingabetyp
 #
-VariableTypesMatchRule.unexpectedType=Validierungsfehler ({0}) : Der Variablentyp ''{1}'' stimmt nicht mit dem erwarteten Typ ''{2}'' �berein
```

Comment:
FYI: GitHub's diff viewer can't handle non-ASCII characters. It was a lower case u with umlaut